browser/synbiohub.js

```diff
@@ -141,7 +141,7 @@ if (typeof meta !== 'undefined') {
             type: 'GET',
             data: function (d) {
                 d.type = 'collectionMembers'
-                d.collectionUri = meta.uri
+                d.collectionUri = meta.remote?meta.uri.toString().replace("/1","/current"):meta.uri
```

Comment:
Can you reformat the ternary so that there are spaces between the operands? Did this not raise a linting error?

Reply:
Done.  No linting did not complain about it.

Reply:
Oh, right, it doesn't link the browser directory.